modules/profile/variables.tf

```diff
@@ -0,0 +1,40 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "instance_id" {
+  type        = string
+  description = "The ID of the SCC instance in a particular region."
+}
+
+variable "profile_name" {
+  type        = string
+  description = "The profile name. Constraints: The maximum length is `64` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "profile_description" {
+  type        = string
+  description = "The profile description. Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "control_libraries" {
+  type = list(object({
+    control_library_name    = optional(string)
+    control_library_version = optional(string)
+  }))
+  default     = []
+  description = "The list of control_library_ids that are used to create the profile. Constraints: The maximum length is `600` items. The minimum length is `0` items."
+}
+
+variable "default_parameters" {
+  type = list(object({
+    assessment_type         = optional(string)
+    assessment_id           = optional(string)
+    parameter_name          = optional(string)
+    parameter_default_value = optional(string)
+    parameter_display_name  = optional(string)
+    parameter_type          = optional(string)
+  }))
+  default     = []
+  description = "The default parameters of the profile. Constraints: The maximum length is `512` items. The minimum length is `0` items."
```

Comment:
This is confusing. My understanding of the way this works is that some of the assessments (rules) belonging to a control support having parameters. So if a user is adding a control to their profile with an assessment (rule) that supports having parameters, they can use this variable to set what the parameters are (and what the default one is). 
I also think that the assessment (rule) itself will have a set of default parameters, so if not explicitly specified here, the defaults are used.

Reply:
most of these descriptions are taken directly from their [docs](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/scc_profile#default_parameters). I updated it to the wording of the ui when you are inputting params.